### PR TITLE
Update investigate-alerts.md

### LINKED ISSUE #12879: Outdated instructions for Connecting Power Automate to Defender for Cloud
The instructions & video for connecting Power Automate to Defender for Cloud Apps are outdated. They are based on using: portal.cloudappsecurity.com which can be confusing especially if tenants also have "Redirection to Microsoft 365 Defender" enabled. 

Current Instructions are based on using - portal.cloudappsecurity.com: 
"Click Settings, select Security extensions, and then click Add token in the API tokens tab."

The new location for this settings in: https://security.microsoft.com
"Click Settings, Click Cloud Apps, Click API tokens and then select Add token in the API tokens tab."

Ideally, both instructions and video would be updated or at least a note where the settings are depending on if you are in:
https://portal.cloudappsecurity.com
or
https://security.microsoft.com

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: a541e506-e227-1fda-e6db-7e8441098753
* Version Independent ID: c1d7febc-d327-85ce-a341-8552617b74d8
* Content: [Investigate alerts in Microsoft 365 Defender](https://learn.microsoft.com/en-us/microsoft-365/security/defender/investigate-alerts?view=o365-worldwide#connect-power-automate-to-microsoft-defender-for-cloud-apps)
* Content Source: [microsoft-365/security/defender/investigate-alerts.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender/investigate-alerts.md)
* Service: **microsoft-365-security**
* Sub-service: **m365d**
* GitHub Login: @diannegali
* Microsoft Alias: **diannegali**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12879

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/investigate-alerts.md [hunk 1]
```diff
@@ -338,7 +338,7 @@ If both are true, SecOps marks the alert as legitimate travel and resolves it. A
 
 To create the automation, you'll need an API token before you can connect Power Automate to Microsoft Defender for Cloud Apps.
 
-1. Click **Settings**, select **Security extensions**, and then click **Add token** in the **API tokens** tab.
+1. Open [Microsoft Defender](https://security.microsoft.com/), Click **Settings**, select **Cloud Apps**, Click **API token** and then Click on **Add token** in the **API tokens** tab.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/investigate-alerts.md]:
```suggestion
1. Open [Microsoft Defender](https://security.microsoft.com/) and select **Settings** > **Cloud Apps** > **API token**, and then select **Add token** in the **API tokens** tab.
```

File: microsoft-365/security/defender/investigate-alerts.md [hunk 2]
```diff
@@ -338,7 +338,7 @@ If both are true, SecOps marks the alert as legitimate travel and resolves it. A
...
 
 2. Provide a name for your token, and then click **Generate**. Save the token as you'll need it later.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/investigate-alerts.md]:
```suggestion
2. Provide a name for your token, and then select **Generate**. Save the token as you'll need it later.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@diannegali Please merge this PR. Thanks

**Pull request merged**